tests/conftest.py

```diff
@@ -149,7 +149,7 @@ def unwind_params(params):
         yield None
         return
 
-    for _ in product(*(((k, v) for v in vs) for k, vs in params.items())):
+    for _ in product(*([(k, v) for v in vs] for k, vs in params.items())):
```

Comment:
It's probably best to pull this change out so that we can backport it to other release branches independently